antsibull/data/docsite/plugins_by_collection.rst.j2

```diff
@@ -12,11 +12,16 @@ These are the plugins in the @{collection_name}@ collection
 
 {% for category, plugins in plugin_maps.items() | sort %}
 
+{% if category == 'module' %}
+Modules
+-------
+{% else %}
 @{ category | capitalize }@ Plugins
```

Comment:
Do we want to specialcase modules like this?

Reply:
Good question. I find "module plugin" a bit confusing, since they were always called "modules" in 2.9 or earlier, and also in the 2.10 docs (except in the antsibull-docs generated parts). On the other hand, the ansible-base codebase makes them less special, at least for collections (plugins/modules/ directory).

I'm leaning a bit to +1 for special-casing modules, but I can also live with not doing it :)

Reply:
We did talk about this in today's docs meeting, the consensus was that we should use 'modules' instead of 'module plugins' (and be consistent with that).